fhir-flow/src/main/java/com/ibm/fhir/flow/api/ResourceIdentifierVersion.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.flow.api;
+
+import java.util.Objects;
+
+import com.ibm.fhir.model.type.Uri;
+import com.ibm.fhir.model.util.ModelSupport;
+
+/**
+ * A resource type, id and version tuple
+ */
+public class ResourceIdentifierVersion extends ResourceIdentifier {
+    // the resource meta.versionId value
+    private final int version;
```

Comment:
do we rely on this being an integer?
for our server it always will be, but its not technically a spec requirement (i.e. other servers could potentially use opaque version ids).  wasn't sure if we only care about our server or wanted to be more general.

Reply:
I'm so used to think of version as an integer...in this case it doesn't actually matter to us because we only use the version for VREAD, so I guess we should keep it as a string.